### DISCUSSION TITLE
Problem in ESXI 8.0.3

### DISCUSSION THREAD

**Discussion by bigmoveus** [w2c/letsencrypt-esxi#37] [General]
Hi getting this error possible any comment ?

[root@hostingdunyam:~] /etc/init.d/w2c-letsencrypt start
Running 'start' action
/opt/w2c-letsencrypt/renew.sh: /opt/w2c-letsencrypt/renew.cfg: line 1: equest: not found
Starting certificate renewal.
Existing cert for hostingdunyam.srp.services not issued by Let's Encrypt. Requesting a new one!
Serving HTTP on :: port 8120 (http://[::]:8120/) ...
Parsing account key...
Parsing CSR...
Found domains: hostingdunyam.srp.services
Getting directory...
Directory found!
Registering account...
Registered! Account ID: https://acme-staging-v02.api.letsencrypt.org/acme/acct/159542293
Creating new order...
Order created!
Verifying hostingdunyam.srp.services...
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:27] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:29] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:30] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:30] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:31] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
::ffff:127.0.0.1 - - [15/Aug/2024 16:29:31] "GET /.well-known/acme-challenge/Bvs5hnd4IXwI7nz07qKKz2ettzI_jCV47TUElp5k52c HTTP/1.1" 200 -
hostingdunyam.srp.services verified!
Signing certificate...
Certificate signed!
Success: Obtained and installed a certificate from Let's Encrypt.
usage: clusterAgent [-h] ACTION
clusterAgent: error: the following arguments are required: ACTION
usage: esxio-commd [-h] ACTION
esxio-commd: error: the following arguments are required: ACTION
logger: Invalid PID 'Usage: fsvmsockrelay '
logger: Invalid PID '{start|stop|status|restart} [--vmci VMCI_ID]'
usage: gpuManager [-h] ACTION
gpuManager: error: the following arguments are required: ACTION
usage: hbrsrv [-h] ACTION
hbrsrv: error: the following arguments are required: ACTION
hostd signalled.
usage: infravisor [-h] ACTION
infravisor: error: the following arguments are required: ACTION
VMware HTTP reverse proxy signalled.
sfcbd-init[2100694]: args ('')
sfcbd-init[2100694]: Getting Exclusive access, please wait...
sfcbd-init[2100694]: Exclusive access granted.
sfcbd-init[2100705]: args ('ssl_reset')
sfcbd-init[2100705]: Getting Exclusive access, please wait...
sfcbd-init[2100705]: Exclusive access granted.
sfcbd-init[2100705]: sfcbd is not running.
logger: Invalid PID 'Usage: vdfsd '
logger: Invalid PID '{start|stop|status|restart|'
vpxa signalled.
vsanperfsvc is not running.
/etc/init.d/vvold ssl_reset, PID 2100819
vvold is not running.

**Comment by Churro** [comment]
Not related to the script specifically but showing when reloading services. Most likely completely without any practical effect.
